client/components/Header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+class Header extends React.Component{
+render(){
+    return(
+        <div className = 'Header'>
+        <h1><p>Name</p></h1>
+        <h2><p>Surname</p></h2>
+        <h3><p>Harakeke2019</p></h3>       
+        </div>
+    )
+}
+}
+
+
+
+
```

Comment:
Why so many extra lines? 😜

Reply:
Omg im so extra !!!!!!! this has been resolved through this commit : https://github.com/kahikatea-2019/MOA/pull/32/commits/07598a4f394b4bb296ffb68d678d6edcddd4ee3b